### PR TITLE
Haml, Boostrap, login & landing 

### PR DESCRIPTION
**Pull request by andresgutgon**
## What?
These changes:
- [x] Use haml as template language
- [x] Use [Font-Awesome](http://fortawesome.github.io/Font-Awesome/) as font for icons
- [x] Use [Autoprexer](https://github.com/ai/autoprefixer-rails) to add browser vendor prefixes
- [x] Use boostrap-sass
- [x] Update login styles
- [x] Port static landing page to rails

## Login
Started playing with login page
image

## Landing versions
image

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,6 +17,8 @@ gem 'pundit'
 gem 'foreman'
 gem 'puma'
 
+gem 'haml'
```
Comment (enricostano) [inline Gemfile]: We should start to use fixed gem versions, no `~`

**Comment by enricostano** [comment]
:clap: looks already awesome! :clap:

**Comment by andresgutgon** [comment]
Maybe we should start translating the page to spanish. Not english. I'll change the file. 
What do you think?

File: engines/account/app/controllers/account/session_controller.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 module Account
   class SessionController < ApplicationController
+    helper Shared::ApplicationHelper
```
Comment (enricostano) [inline engines/account/app/controllers/account/session_controller.rb]: You have it already in `Account::ApplicationController`, right?

**Comment by enricostano** [comment]
@andresgutgon I see the images a bit blurred, why?

**Comment by andresgutgon** [comment]
Ready to merge :tada: 

@enricostano if you want to change something before merge. Feel free to make a commit. If not :couplekiss:

File: engines/account/app/views/account/signup/complete.html.haml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+- if flash[:error]
+  %p= flash[:error]
+#signupbox.mainbox.col-md-6.col-md-offset-3.col-sm-8.col-sm-offset-2{style: "margin-top:50px;"}
+  .panel.panel-info
+    .panel-heading
+      .panel-title
+        Complete sign up for
```
Comment (naoisegolden) [inline engines/account/app/views/account/signup/complete.html.haml]: Hardcoded text :speak_no_evil:

File: engines/account/app/views/account/signup/email_sent.html.haml [hunk 4]
```diff
@@ -0,0 +1 @@
+We sent you an email at #{@signup.email}
```
Comment (naoisegolden) [inline engines/account/app/views/account/signup/email_sent.html.haml]: Idem hardcoded text.

File: engines/landing/app/assets/stylesheets/landing/application.sass [hunk 5]
```diff
@@ -0,0 +1,136 @@
+@import "../../../../../shared/app/assets/stylesheets/shared/variables"
+@import "../../../../../shared/app/assets/stylesheets/shared/katuma-bootstrap/variables"
+
+// @roboto: "Roboto:300,400";
+// @roboto-slab: "Roboto Slab";
+// @url1: "http://fonts.googleapis.com/css?family=@{roboto}";
+// @url2: "http://fonts.googleapis.com/css?family=@{roboto-slab}";
+// @import url(@url1);
+// @import url(@url2);
```
Comment (naoisegolden) [inline engines/landing/app/assets/stylesheets/landing/application.sass]: Can these be deleted?

File: engines/landing/config/locales/ca.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+ca:
+  landing:
+    slogan: AEstalvia't l'intermediari, compra directament als productors.
```
Comment (naoisegolden) [inline engines/landing/config/locales/ca.yml]: typo

**Comment by enricostano** [comment]
@andresgutgon I still see favicon related issues running the server:
```
Started GET "/favicon-16x16.png%22%20sizes=%2216x16" for 127.0.0.1 at 2015-08-06 20:57:48 +0200

ActionController::RoutingError (No route matches [GET] "favicon-16x16.png%22%20sizes=%2216x16"):


Started GET "/favicon-96x96.png%22%20sizes=%2296x96" for 127.0.0.1 at 2015-08-06 20:57:48 +0200

ActionController::RoutingError (No route matches [GET] "favicon-96x96.png%22%20sizes=%2296x96"):


Started GET "/android-chrome-192x192.png%22%20sizes=%22192x192" for 127.0.0.1 at 2015-08-06 20:57:47 +0200

ActionController::RoutingError (No route matches [GET] "android-chrome-192x192.png%22%20sizes=%22192x192"):

```

File: engines/account/app/views/account/signup/email_sent.html.haml (hunk 4 above)
Comment (enricostano) [inline engines/account/app/views/account/signup/email_sent.html.haml]: My fault, I'll fix it in next PR

File: engines/account/app/views/account/signup/complete.html.haml (hunk 3 above)
Comment (enricostano) [inline engines/account/app/views/account/signup/complete.html.haml]: My fault, I'll fix it in next PR

**Comment by enricostano** [comment]
Really great job @andresgutgon :clap: :smiley:

**Pull request merged**